src/Symfony/Component/Workflow/Registry.php

```diff
@@ -25,13 +26,17 @@ class Registry
     /**
      * @param Workflow                 $workflow
      * @param SupportStrategyInterface $supportStrategy
+     *
+     * @deprecated since Symfony 4.1. Use addWorkflow() instead.
```

Comment:
`@deprecated since version 4.1, to be removed in 5.0. Use addWorkflow() instead.`